### ISSUE TITLE
Delete learned words

### ISSUE THREAD

**Issue by pudup** [dslul/openboard#106]
As of right now, with word learning turned on, accidentally selecting a word as it's misspelled adds it to the internal dictionary and then starts being used for autocorrection.

There currently isn't, as far as I can tell, a way to remove this learned word or words without turning off 'learn new words' completely.

Perhaps a long press function to delete learned words would be good.

**Comment by TheCrowned** [comment]
Duplicate of #28

**Comment by dslul** [comment]
I think they are different, that issue refers to word by word deletion when pressing delete for a long time.

**Comment by jwsp1** [comment]
Since openboard saves every single word you write, it will also likely save typos. The need to delete learned words would not be so big if openboard only learns words that have been written 3 times or so. But for now I don't recommend changing from 1 to 3 until #143 (sugest next words from a common dictionary) is solved because with 3 you don't get even less suggestions for next words.

**Comment by m1chj** [comment]
This feature is the only thing OpenBoard is missing

**Comment by nach0** [comment]
I am on the same boat: OpenBoard has learned some of my common spelling mistakes and now uses them as proper words. An option to see the learned words and edit them would be very much appreciated. Thanks :)

**Comment by SilverPuppy** [comment]
I do not think this is a duplicate of #368 as that one is asking for a menu option to review and modify the learned word list, where this asks for a long press option to delete the learned word. I really do not wish to reset my board completely, but I'm very tired of constantly getting mistakes as suggestions. I'd like to see this request and #368 implemented, but if either one does, I'll be much happier!
